TheaterDatabase/TheaterDatabase.sqlproj

```diff
@@ -64,25 +64,15 @@
   </ItemGroup>
   <ItemGroup>
     <Build Include="dbo\Tables\Account.sql" />
-    <Build Include="dbo\Tables\CreativeTeamMember.sql" />
     <Build Include="dbo\Tables\GalleryImage.sql" />
```

Comment:
Do we still loading images from database?
this should be done using wordpress api
I think this should also be removed. please check usages